Book-A-Desk.Api/EmailNotification.fs

```diff
@@ -0,0 +1,46 @@
+namespace Book_A_Desk.Api
+
+open System
+open System.Net.Mail
+open Book_A_Desk.Api.Models
+open Book_A_Desk.Domain.QueriesHandler
+open Book_A_Desk.Domain.Office.Domain
+
+type EmailNotification =
+    {
+        SendEmailNotification: Booking -> unit
+    }
+
+module EmailNotification =
+    let initialize getEmailServiceConfiguration getOffices =         
+        let sendEmailNotification (booking: Booking) =
+            let config = getEmailServiceConfiguration()        
+            let officeName =
+                let result = OfficeQueriesHandler.getAll getOffices
+                let officeReference = Guid.Parse(booking.Office.Id) |> OfficeId
+                match result with
+                | Ok offices ->
+                    offices
+                    |> List.find (fun (o:Office) -> officeReference.Equals(o.Id))
+                    |> (fun (o:Office) -> 
+                        let (CityName cityName) = o.City
+                        cityName)
+                | Error e ->
+                    "Unknown"
+                    
+            use smtpClient = new SmtpClient(config.SmtpClientUrl)
+            smtpClient.EnableSsl <- true
+            smtpClient.Credentials <- System.Net.NetworkCredential(config.SmtpUsername, config.SmtpPassword)            
+            use mailMessage =
+                new MailMessage(
+                    config.EmailSender,
+                    booking.User.Email)
+            mailMessage.CC.Add config.EmailReviewer
+            mailMessage.Subject <- "Book-A-Desk Reservation confirmed"
+            mailMessage.Body <- $"You have booked a desk at %s{booking.Date.ToShortDateString()} in the Office %s{officeName}"
+
+            smtpClient.Send(mailMessage)
```

Comment:
Can this be done async? It's usually better to do things async if they're available, since it doesn't freeze up a thread while we're waiting

Reply:
It can, we just did it this way for simplicity with the temporary solution.

Reply:
I don't think the async solution is more complicated though. The handler that calls this is already async, so it's most likely just a matter of doing smtpClient.SendAsync()

Reply:
To Do: Adding async management.